src/Compilers/CSharp/Portable/CSharpCompilationOptions.cs

```diff
@@ -83,49 +86,6 @@ public sealed class CSharpCompilationOptions : CompilationOptions, IEquatable<CS
         {
         }
 
-        // 15.6 BACKCOMPAT OVERLOAD -- DO NOT TOUCH
-        [EditorBrowsable(EditorBrowsableState.Never)]
-        public CSharpCompilationOptions(
-            OutputKind outputKind,
-            bool reportSuppressedDiagnostics,
-            string moduleName,
-            string mainTypeName,
-            string scriptClassName,
-            IEnumerable<string> usings,
-            OptimizationLevel optimizationLevel,
-            bool checkOverflow,
-            bool allowUnsafe,
-            string cryptoKeyContainer,
-            string cryptoKeyFile,
-            ImmutableArray<byte> cryptoPublicKey,
-            bool? delaySign,
-            Platform platform,
-            ReportDiagnostic generalDiagnosticOption,
-            int warningLevel,
-            IEnumerable<KeyValuePair<string, ReportDiagnostic>> specificDiagnosticOptions,
```

Comment:
we use KVP in one place, but tuples in another... i have no good answer for what we should be doing.  but it seems pretty inconsistent.

Reply:
I'll just removed the IEnumerable and force it to be ImmutableDictionary. I don't know why we allow enumerables anyway.